### PR TITLE
FSA Spotter Changes

### PR DESCRIPTION
**Pull request by naveenmadhire**

### CODE REVIEW

File: core/src/test/scala/org/dbpedia/spotlight/db/spotter/TestSfFSASpotter.scala [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+  val Sfs = List("agents that cause birth defects","Controversies","causes","diagnostic criteria","brain","nerve cell","synapse","autism spectrum","Asperger syndrome","pervasive developmental disorder, not otherwise specified")
```
Comment (dav009) [inline core/src/test/scala/org/dbpedia/spotlight/db/spotter/TestSfFSASpotter.scala]: Can split this into multiple lines just for readability

File: core/src/main/scala/org/dbpedia/spotlight/db/tokenize/LanguageIndependentTokenizer.scala [hunk 2]
```diff
@@ -44,6 +44,8 @@ class LanguageIndependentTokenizer(
         token
       }
     }.flatten.toList
+
```
Comment (dav009) [inline core/src/main/scala/org/dbpedia/spotlight/db/tokenize/LanguageIndependentTokenizer.scala]: empty lines

File: core/src/main/scala/org/dbpedia/spotlight/db/FSASpotter.scala [hunk 3]
```diff
@@ -14,15 +14,67 @@ import tokenize.LanguageIndependentTokenizer
...
+  private var name = "FSA dictionary spotter"
+  def getName = name
+  def setName(name: String) {
+    this.name = name
+  }
+
+}
+
+class AllOccurrencesFSASpotter(
+                                fsaDictionary: FSADictionary,
```
Comment (dav009) [inline core/src/main/scala/org/dbpedia/spotlight/db/FSASpotter.scala]: style

**Comment by dav009** [comment]
It seems it is building correctly, the error on CI is that it couldn't fetch a dep.

I will close and then reopen the PR so it gets rebuilt on CI.

**Comment by naveenmadhire** [comment]
Not sure. Intellij IDE which I am using is changing the formatting on the existing files.

**Pull request merged**